### PR TITLE
rokt destination docs

### PR DESCRIPTION
**Pull request by tcgilbert**
### Proposed changes

add docs for new destination

### Merge timing
asap

### Related issues (optional)
n/a

### CODE REVIEW

File: src/connections/destinations/catalog/rokt-integration/index.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+title: Rokt Integration Destination
+id: 6268a16ce311a548d8cb1a72
+---
+
+The Rokt-Segment destination optimizes campaign performance by integrating conversion attribution for Rokt Ads. By integrating this conversion data, you’ll unlock a suite of intelligent tools that learn from every conversion, constantly making adjustments to improve campaign targeting and bidding.
```
Comment (stayseesong) [inline src/connections/destinations/catalog/rokt-integration/index.md:6]:
```suggestion
The Rokt destination optimizes campaign performance by integrating conversion attribution for Rokt Ads. By integrating this conversion data, you’ll unlock a suite of intelligent tools that learn from every conversion, constantly making adjustments to improve campaign targeting and bidding.
```

File: src/connections/destinations/catalog/rokt-integration/index.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+This destination is maintained by Rokt. If you have any issues, please contact [Rokt support team](https://rokt.atlassian.net/servicedesk/customer/portal/20/group/98).
```
Comment (stayseesong) [inline src/connections/destinations/catalog/rokt-integration/index.md:8]:
```suggestion
This destination is maintained by Rokt. If you have any issues, please contact the [Rokt support team](https://rokt.atlassian.net/servicedesk/customer/portal/20/group/98){:target="_blank"}.
```

File: src/connections/destinations/catalog/rokt-integration/index.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+## Getting Started
+
+1. From the Destinations catalog page in the Segment App, click **Add Destination**.
+2. Search for “Rokt” in the Destinations Catalog. Select the “Rokt Integration” destination.
```
Comment (stayseesong) [inline src/connections/destinations/catalog/rokt-integration/index.md:13]:
```suggestion
2. Search for “Rokt” in the Destinations Catalog, and select the integration.
```

File: src/connections/destinations/catalog/rokt-integration/index.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+3. Choose which source should send data to the “Rokt Integration” destination.
```
Comment (stayseesong) [inline src/connections/destinations/catalog/rokt-integration/index.md:14]:
```suggestion
3. Choose which source should send data to the Rokt destination.
```

File: src/connections/destinations/catalog/rokt-integration/index.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+4. Enter the API key provided to you by your Rokt Account manager. If you haven't received your login credentials, please reach out to them.
+
+Once you've entered the API credentials for Rokt, the chosen source will send data through to Rokt's API.
```
Comment (stayseesong) [inline src/connections/destinations/catalog/rokt-integration/index.md:17]:
```suggestion
Once you've entered the API credentials for Rokt, the chosen source sends data through to Rokt's API. 
```

File: src/connections/destinations/catalog/rokt-integration/index.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+
+Once you've entered the API credentials for Rokt, the chosen source will send data through to Rokt's API. 
+
+## Supported Methods
+
+Rokt supports the `track` API call, which can be used to send data to Rokt:
+
+
+### Track
+The `track` API call is how you record any actions your users perform, along with any properties that describe the action. Read more [here](https://segment.com/docs/connections/spec/track/).
```
Comment (stayseesong) [inline src/connections/destinations/catalog/rokt-integration/index.md:25]:
```suggestion
The `track` API call is how you record any actions your users perform, along with any properties that describe the action. [Learn more about the track call](/docs/connections/spec/track/). 
```

**Review by stayseesong — CHANGES_REQUESTED** [review]

File: src/connections/destinations/catalog/rokt-integration/index.md (hunk 3 above)
Comment (tcgilbert) [reply src/connections/destinations/catalog/rokt-integration/index.md:13]: @stayseesong so I agree that "Rokt Integration" does not flow well, but this is actually the name for this destination on our backend. Meaning they will see "Rokt Integration"  show up when searching for this in a segment workspace. 

With that in mind I think we should keep it as is, but I am open to suggestions.

Comment (markzegarelli) [reply src/connections/destinations/catalog/rokt-integration/index.md:13]: @tcgilbert Do you know if this naming was intentional? It does not match the format of other integration names in the system. I agree that the article should match the actual name of the destination, but we may want to suggest a rename here.

Comment (markzegarelli) [reply src/connections/destinations/catalog/rokt-integration/index.md:13]: @tcgilbert Got it, thanks for the background. I don't want to delay this for docs reasons, was just curious.

Comment (tcgilbert) [reply src/connections/destinations/catalog/rokt-integration/index.md:13]: @markzegarelli Gotcha. yeah they are keen on launching this sooner than later. 

we can go ahead and go with this name, and if they want to change in the future, I can come back and update this.

**Review by stayseesong — APPROVED** [review]

**Pull request merged**